src/blocks/server.blocks/middleware/__router/middleware__router.node.js

```diff
@@ -56,6 +56,14 @@ modules.define(
             // remove level suffixes
             // Remove tab parts of url before processing it
             // Remove trailing slash for url
+            var tabSuffix = url.match(/\/docs\/?$/);
```

Comment:
Previous version contain code about you say.
@bemer can you to finish this PR ?